tree_migration_milestone1.js

```diff
@@ -71,9 +73,18 @@ async function migrate() {
       };
     };
 
-    for await (const tree_object of query_stream) {
+   
+    ws._write = async (tree_object, enc, next) => {
```

Comment:
using `Writable` and `query_stream.pipe(ws)` allows us to not depend on `for await`.